ARCADEGC/Card-Game
# Implemented Howler with Test Aduio

# Implemented Howler with Test Audio

## Description

This pull request implements the Howler library to handle audio playback in the application. It includes a test audio file that can be used for verifying the functionality.

The key changes include:

- Integrated the Howler library to manage audio playback
- Added a test audio file to the project for verification purposes
- Implemented the necessary logic to play the test audio using the Howler library

## Testing

To test the audio playback functionality, please follow these steps:

1. Ensure the test audio file is present in the project
2. Run the application and verify that the test audio can be played successfully

## Checklist

- [x] Integrated Howler library for audio playback
- [x] Added test audio file to the project
- [x] Implemented logic to play the test audio
- [x] Verified the audio playback functionality
